src/components/GroupsSelect/SelectBox/Control.jsx

```diff
@@ -6,23 +6,30 @@ import Icon from 'cozy-ui/transpiled/react/Icon'
 import palette from 'cozy-ui/transpiled/react/palette'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 
-const Control = ({ selectProps: { onControlClicked } }) => {
+const Control = ({
+  innerRef,
+  innerProps,
+  selectProps: { onControlClicked }
+}) => {
   const { t } = useI18n()
 
   return (
-    <Button
-      theme="secondary"
-      size="small"
-      label={t('groups.manage')}
-      onClick={onControlClicked}
-    >
-      <Icon
-        icon="bottom-select"
-        color={palette['coolGrey']}
-        width="12"
-        className="group-manager__indicator"
-      />
-    </Button>
+    <div ref={innerRef} {...innerProps}>
+      <Button
+        theme="secondary"
+        size="small"
+        label={t('groups.manage')}
+        onClick={onControlClicked}
+        onTouchStart={onControlClicked}
+      >
+        <Icon
+          icon="bottom-select"
+          color={palette['coolGrey']}
+          width="12"
+          className="group-manager__indicator"
+        />
```

Comment:
Nit: I think the Button has an `icon` prop that'd be useful here, it is a bit weird otherwise to understand that the Icon children will be next to the label prop.

Reply:
`icon` ne permet que d'afficher une icône à gauche du texte. Pour la droite il faudrait utiliser `extraRight` mais wrap le contenu dans un span qui vient avec son css hérité, ce qui demanderait de revoir le css ici dans le cas de Contacts.

Reply:
OK in this case I think it would be a little bit clearer that the label was part of the children. It was not immediately clear that the icon was to the right.